tools/autograd/templates/VariableType.cpp

```diff
@@ -335,26 +335,26 @@ static bool isFloatingPoint(ScalarType s) {
   return s == kFloat || s == kDouble || s == kHalf;
 }
 
-void VariableType::s_copy(const Tensor & src, Tensor & dst) const {
+Tensor & VariableType::s_copy_(Tensor & self, const Tensor & src, bool async) const {
```

Comment:
uh are we changing src-dst order again...

Reply:
I'm making it consistent with every other method in Type